src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,70 @@
+package com.kodilla.ecommercee.domain;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "products")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Product {
+
+    @Column(name = "id")
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(name = "name")
+    @NotNull
+    private String name;
+
+    @Column(name = "price")
+    @NotNull
+    private double price;
+
+    @Column(name = "description")
+    private String description;
+
+    @ManyToOne
+    @JoinColumn(name = "group_id")
+    private Group productGroup;
+
+    @ManyToOne
+    @JoinColumn(name = "cart_id")
+    private Cart cart;
+
+    public Product(@NotNull String name, @NotNull double price, String description, Group productGroup, Cart cart) {
```

Comment:
Remove all those constructors. Use builder pattern instead
```suggestion
    public Product(@NotNull String name, @NotNull double price, String description, Group productGroup, Cart cart) {
```